apps/src/webapp/levels.js

```diff
@@ -42,6 +42,8 @@ levels.ec_simple = {
     {'func': 'setParent', 'title': 'Set an element to become a child of a parent element', 'category': 'General', 'params': ["'id'", "'parentId'"] },
     {'func': 'setPosition', 'title': 'Position an element with x, y, width, and height coordinates', 'category': 'General', 'params': ["'id'", "0", "0", "100", "100"] },
     {'func': 'setStyle', 'title': 'Add CSS style text to an element', 'category': 'General', 'params': ["'id'", "'color:red;'"] },
+    {'func': 'getAttribute', 'category': 'General', 'params': ["'id'", "'innerHTML'"], 'type': 'value' },
```

Comment:
rather than innerHTML as the example attribute name, can you think of something more common? Maybe scrollHeight like you had to use in your sample. My worry is that we encourage students to party on innerHTML, but maybe they will need to do that.